src/text-editor.coffee

```diff
@@ -677,7 +677,7 @@ class TextEditor extends Model
   # this editor.
   shouldPromptToSave: ({windowCloseRequested}={}) ->
     if windowCloseRequested
-      @isModified()
+      false
```

Comment:
Based on #942, we now avoid to prompt the user when a window close request is in place. What are your thoughts on this, @atom/feedback? :thought_balloon: